docs/release_notes/v1.2.0.md

```diff
@@ -0,0 +1,271 @@
+- **RESOLVED** Outdated instructions for running Kafka on Kubernetes [1327](https://github.com/dapr/docs/issues/1327)
+- **RESOLVED** chore: add documentation on the binding mqtt based on the pubsub mqtt… [1328](https://github.com/dapr/docs/pull/1328)
+- **RESOLVED** [CONTENT] PubSub add more redis client settings [1339](https://github.com/dapr/docs/issues/1339)
+- **RESOLVED** Update HashiCorp Vault secret docs [1354](https://github.com/dapr/docs/issues/1354)
+- **RESOLVED** Update hashicorp-vault.md [1355](https://github.com/dapr/docs/pull/1355)
+- **RESOLVED** Add documentation for PowerShell and Command Prompt [1373](https://github.com/dapr/docs/issues/1373)
+- **RESOLVED** Remove workaround for mTLS bug [1387](https://github.com/dapr/docs/issues/1387)
+- **RESOLVED** [CONTENT] Add CLI reference docs for `--data-file` flags for invoke and publish [1391](https://github.com/dapr/docs/issues/1391)
+- **RESOLVED** Add Alibaba Cloud Nacos binding Doc [1401](https://github.com/dapr/docs/pull/1401)
+- **RESOLVED** Add Alibaba Cloud RocketMQ pubsub Doc [1405](https://github.com/dapr/docs/pull/1405)
+- **RESOLVED** Add Alibaba Cloud RocketMQ binding doc [1406](https://github.com/dapr/docs/pull/1406)
+- **RESOLVED** Add Alibaba Cloud DingTalk Webhook binding doc [1407](https://github.com/dapr/docs/pull/1407)
+- **RESOLVED** [CONTENT] Consul nameresolution component [1408](https://github.com/dapr/docs/issues/1408)
+- **RESOLVED** sentinel middleware doc  [1439](https://github.com/dapr/docs/pull/1439)
+- **RESOLVED** Add documentation for the Zeebe bindings [1441](https://github.com/dapr/docs/pull/1441)
+- **RESOLVED** Allow choosing which APIs are available on a sidecar  [1460](https://github.com/dapr/docs/issues/1460)
+- **RESOLVED** v1.2 Release [1461](https://github.com/dapr/docs/issues/1461)
+- **RESOLVED** Documentation for Preview Features [1463](https://github.com/dapr/docs/issues/1463)
+- **RESOLVED** Document how to set the default values for actors [1470](https://github.com/dapr/docs/issues/1470)
+- **RESOLVED** [CONTENT] Document pubsub without CE for 1.2 [1474](https://github.com/dapr/docs/issues/1474)
+- **RESOLVED** Create a shareable Dapr PowerPoint deck [1477](https://github.com/dapr/docs/issues/1477)
+- **RESOLVED** [CONTENT] Add CLI reference docs for `--dapr-http-max-request-size` flags for run [1481](https://github.com/dapr/docs/issues/1481)
+- **RESOLVED** Add Docs for Actor Reentrancy [1482](https://github.com/dapr/docs/issues/1482)
+### Kit
+- **MOVED** logger package from `dapr/dapr` to `dapr/kit` [1](https://github.com/dapr/kit/pull/1)
+### Homebrew-tap
+- **REPLACED** deprecated formula style with current alternative [20](https://github.com/dapr/homebrew-tap/pull/20)
+
+## Upgrading to Dapr 1.2.0
+
+To upgrade to this release of Dapr, follow the steps here to ensure a smooth upgrade. You know, the one where you don't get red errors on the terminal.. we all hate that, right?
+
+
```

Comment:
Do we need extra empty lines here?

Reply:
Probably not. Remember this file is auto-generated from the template. We'll have to change it in the template also. I can manually change it here, but if we don't change the template we will have to make the same changes for future release notes.

Reply:
Even if I remove the spaces it still renders the same. I'll remove them, but it won't look any different!